src/Propel/Generator/Model/Database.php

```diff
@@ -842,7 +842,7 @@ public function __toString()
             foreach ($table->getColumns() as $column) {
 
                 $columns[] = sprintf("      %s %s %s %s %s %s %s",
-                    $column->getName(),
+                    $column->getName(), 
```

Comment:
use it for restart travis checks, it some times get false-positive errors, like that https://github.com/propelorm/Propel2/pull/1377/commits/dc24753acac6cd83c1ec03a36e85449535568159